CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 1.4.20 (Nov 2020)
+  * XCode 12 support
+  * Completely reworked escape analysis for object allocation
+  * Use ForeignException wrapper to handle native exceptions ([KT-35056](https://youtrack.jetbrains.com/issue/KT-35056))
+  * CocoaPods plugin improvements
+  * equals/hashCode support for adapted callable references ([KT-39800](https://youtrack.jetbrains.com/issue/KT-39800))
+  * equals/hashCode support for fun interfaces ([KT-39798](https://youtrack.jetbrains.com/issue/KT-39798))
+  * IR-level optimizations
+    * Constant folding
+    * String concatenation flattenning
+    * Emit Lazy IR for cached structs and enums
```

Comment:
Not sure if this is actually an optimization. I mean, the other two points ^^ optimize the runtime performance, but this one is probably unrelated.